src/main/resources/wa-task-initiation-st_cic-criminalinjuriescompensation.dmn

```diff
@@ -1104,7 +1104,7 @@
           <text></text>
         </outputEntry>
         <outputEntry id="LiteralExpression_1a83obu">
-          <text>1</text>
+          <text>5</text>
```

Comment:
Unrelated but could yo fix the naming on [L:1005](https://github.com/hmcts/st-wa-task-configuration/pull/159/files#diff-066c6baf6f08923d466e5309ab71dffd80da783f60b644240a8e5c10bac6bcf9R1005) to update the task name for `reviewOtherRequestLO` as it's currently "Review Reinstatement request - Legal Officer" and should be "Review Other Request - Legal Officer"